### PR TITLE
Support Django 2.0

### PR DESCRIPTION
**Pull request by Surgo**
Subject: Support Django 2.0

### Feature or Bugfix

- [x] Feature
- [ ] Bugfix

### Relates

- https://docs.djangoproject.com/en/2.0/releases/2.0/

### CODE REVIEW

File: tests/testapp/models.py [hunk 1]
```diff
@@ -23,5 +23,5 @@ class TestParentModel(models.Model):
 
 
 class TestChildModel(models.Model):
-    parent = models.ForeignKey(TestParentModel)
+    parent = models.ForeignKey(TestParentModel, on_delete=models.CASCADE)
```
Comment (Surgo) [inline tests/testapp/models.py]: `on_delete` is always required.

https://docs.djangoproject.com/en/2.0/releases/1.9/#foreignkey-and-onetoonefield-on-delete-argument

File: django_redshift_backend/base.py [hunk 2]
```diff
@@ -159,8 +159,10 @@ def create_model(self, model):
                 definition += " %s" % col_type_suffix
             params.extend(extra_params)
             # FK
-            if field.rel and field.db_constraint:
-                to_table = field.rel.to._meta.db_table
+            if field.remote_field and field.db_constraint:
```
Comment (Surgo) [inline django_redshift_backend/base.py]: `Field.rel` is deprecated in 1.9 and removed in 2.0.

https://docs.djangoproject.com/en/2.0/releases/1.9/#field-rel-changes

File: setup.py [hunk 3]
```diff
@@ -33,10 +33,8 @@ def read(filename):
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Framework :: Django',
-        'Framework :: Django :: 1.8',
-        'Framework :: Django :: 1.9',
-        'Framework :: Django :: 1.10',
         'Framework :: Django :: 1.11',
+        'Framework :: Django :: 2.0',
```
Comment (Surgo) [inline setup.py]: I'm sorry...

File: django_redshift_backend/base.py [hunk 4]
```diff
@@ -159,8 +159,10 @@ def create_model(self, model):
...
+                to_table = field.remote_field.to._meta.db_table
+                to_column = field.remote_field.to._meta.get_field(
+                    field.remote_field.field_name).column
                 to_column = field.rel.to._meta.get_field(field.rel.field_name).column
```
Comment (shimizukawa) [inline django_redshift_backend/base.py]: I think this line must be removed because new code is provided at one line before.

File: setup.py (hunk 3 above)
Comment (shimizukawa) [reply setup.py]: Ah, I also overlooked ;)

File: django_redshift_backend/base.py (hunk 4 above)
Comment (Surgo) [reply django_redshift_backend/base.py]: oh... sorry

**Pull request merged**